packages/@expo/cli/src/start/server/metro/debugging/createDebugMiddleware.ts

```diff
@@ -19,7 +19,7 @@ export function createDebugMiddleware(metroBundler: MetroBundlerDevServer) {
 
   const { middleware, websocketEndpoints } = createDevMiddleware({
     projectRoot: metroBundler.projectRoot,
-    serverBaseUrl: metroBundler.getJsInspectorBaseUrl(),
+    serverBaseUrl: metroBundler.getUrlCreator().constructUrl({ scheme: 'http', hostType: 'lan' }),
```

Comment:
Is host type required here? If it has to be defined, I might need to do some tests (we can still merge it as is). I think I only used `getJsInspectorBaseUrl` as we previously had this in other places: https://github.com/expo/expo/blob/cba26356d763477b4d36b863ab8acfc568fd3a7e/packages/%40expo/cli/src/start/interface/interactiveActions.ts#L94

Reply:
If we don't specify `hostType`, tunnel will be used, and given that ngrok would not be running yet we would 
get the warning thrown 

https://github.com/expo/expo/blob/20cd854e7d73ce27169aa27962cb64c1734ad11f/packages/@expo/cli/src/start/server/UrlCreator.ts#L107-L112

Reply:
Going to merge this in for now because of today's release, but we can adjust this in a follow-up PR if necessary